task2.js

```diff
@@ -0,0 +1,174 @@
+//==================Task1===============
+
+const numbers = [1, 2, 3];
+const numbers2 = [5, 4, 9];
+const changeArrayNumbers = (array) => {
+  return array.map((currentValue) => {
+    return currentValue % 2 === 0 ? currentValue + 1 : currentValue - 1;
+  });
+};
+console.log(changeArrayNumbers(numbers2));
+console.log(changeArrayNumbers(numbers));
+console.log(numbers[1]);
+
+//==================Task2===============
+
+const changeWords = (value) => {
+  return value.map((string) => {
+    return string[0].toUpperCase() + string.substring(1).toLowerCase();
+  });
+};
+
+console.log(changeWords(["asdRadSd", "dDFDSdsf"]));
+
+//==================Task3===============
+
+const users = [
+  {
+    id: 1,
+    name: "Alex",
+  },
+  {
+    id: 2,
+    name: "Tomara",
+  },
+  {
+    id: 3,
+    name: "Max",
+  },
+];
+
+const deleteUsersId = (userList, idList) => {
+  return userList.filter((user) => {
+    return !idList.includes(user.id);
+  });
+};
+console.log(deleteUsersId(users, [2]));
+
+//==================Task4===============
+
+const isPalindrome = (string) => {
+  return string.split("").reverse().join("") === string;
```

Comment:
compare in one register. if first letter will be in upper case, your function will work wrong